lib/apiservers/portlayer/swagger.json

```diff
@@ -1748,6 +1748,63 @@
 				}
 			}
 		},
+		"/containers/{handle}/name": {
+			"put": {
```

Comment:
One more thing, docker doesn't implement full REST api, we simply should not follow their idiom here, it would become a mess.